service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellClient.java

```diff
@@ -31,15 +33,30 @@ public CromwellClient(
     singletonHttpClient = new ApiClient().getHttpClient();
   }
 
-  private ApiClient getApiClient()
+  public ApiClient getWriteApiClient()
       throws DependencyNotAvailableException, AzureAccessTokenException {
+    String uri =
+        dependencyUrlLoader.loadDependencyUrl(DependencyUrlLoader.DependencyUrlType.CROMWELL_URL);
```

Comment:
Maybe we should check `testingBaseUri` here as well so that we can test locally against a single running cromwell?